shared/chat/conversation/normal/index.desktop.tsx

```diff
@@ -79,7 +80,7 @@ const Conversation = React.memo(function Conversation() {
       >
         {threadLoadedOffline && <Offline />}
         <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.innerContainer}>
-          <ListArea />
+          <ListArea key={conversationIDKey} />
```

Comment:
Since we do have internal caching using refs we should just remount fresh if the convo changes so there's no confusion there, just simplify